src/controllers/folder.controller.js

```diff
@@ -11,180 +11,243 @@ exports.getChildren = (req, res) => {
 };
 
 exports.makeFolder = async (req, res) => {
+  let response = {
+    ok: true,
```

Comment:
weren't you going to initialise this to `ok: false` and only change it to `ok: true` only once it was certain the request succeeded?